### ISSUE TITLE
Github: templates 

### ISSUE THREAD

**Issue by mm12** [LAION-AI/Open-Assistant#1652]
The `Issue template`s &  `Pull request template`s will at the very least make labeling easier.

**Comment by andreaskoepf** [comment]
Of course it depends on the specific templates used but in many cases where I saw gh templates they seemed to be a tool of the dark side of the force (personal opinion) .. e.g. they ask people to go through a (in many cases unnecessary and irrelevant) process before submitting their idea, question or bug-report, i.e. they take away freedom and discourage contributions and have very little benefits.

**Comment by mm12** [comment]
> Of course it depends on the specific templates used but in many cases where I saw gh templates they seemed to be a tool of the dark side of the force (personal opinion) .. e.g. they ask people to go through a (in many cases unnecessary and irrelevant) process before submitting their idea, question or bug-report, i.e. they take away freedom and discourage contributions and have very little benefits.

I agree with some of this. The templates can be really bad sometimes, but they are very customizable. ie, here is what each could look like. 

- Bug report: (applies `bug?` label) - adds "link any relevant messages if applicable" at the bottom
- Feat. request: (applies the FR label) - Ask what part of the project it applies to (with 'not sure' as an option)
- Generic issue: Acts like a normal issue
- Question - applies the question label, and has a section for relevant links.

**Comment by bitplane** [comment]
I agree, they can be a barrier to contributing and are used as a way to deflect issues.

But in my experience as the project becomes more mainstream users do need to be told that a good bug report has "what did you do? what did you expect to happen? what actually happened?" parts

**Comment by Nil-Andreu** [comment]
I think that it can be a good idea if the template does not add a lot of required fields. As those ones that are mandatory (in case of a bug for example; which is the and the details if needed to reproduce it) will really help to make those Issues efficient & useful.
